src/styles/blocks/star.scss

```diff
@@ -0,0 +1,19 @@
+.stars {
+  display: flex;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star-active.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-child) {
+    margin-right: 4px;
+  }
+
+  &--4 > :last-child {
+    background-image: url(../images/star.svg);
```

Comment:
Would be great to add other modifiers, so you can easily switch between amount of stars